AshwiniP/strcture.c

```diff
@@ -0,0 +1,28 @@
+#include<stdio.h>
+
+void main()
```

Comment:
1) main should come in next line. 
2) Give 4 character space within a body of loop or if condition.
3) Give 4 character space while declaring the variables.